assets/templates/partials/inputs/input-textarea.tmpl

```diff
@@ -0,0 +1,23 @@
+{{/* Use model.Input for model binding */}}
+<label
+    class="ons-label ons-label--with-description"
```

Comment:
Description is optional here in the service manual, would be good to support.

Reply:
True, I was taking the existing use as a baseline but I can make it optional/conditional